.circleci/scripts/binary_checkout.sh

```diff
@@ -34,11 +34,13 @@ else
 fi
 
 # Clone the Pytorch branch
-retry git clone https://github.com/pytorch/pytorch.git "$PYTORCH_ROOT"
-pushd "$PYTORCH_ROOT"
+retry git clone https://github.com/pytorch/builder.git "$BUILDER_ROOT"
```

Comment:
LOL. Why do you change like this? Isn't it easier to just change line 59 into the following line?
```bash
retry git clone -q https://github.com/mszhanyi/builder.git -b zhanyi/addcu11bintest "$PYTORCH_ROOT"
```

Reply:
Sorry, at the begining, I just want to run a test quickly so didn't take time to walk through the whole script